bundles/org.openhab.binding.metofficedatahub/README.md

```diff
@@ -0,0 +1,463 @@
+| max10mWindGust            | Number:Speed         | Maximum 10m Wind Gust Speed of Previous Hour | m/s  |
+| windDirectionFrom10m      | Number:Angle         | 10m Wind From Direction                      | °    |
+| screenDewPointTemperature | Number:Temperature   | Dew Point Temperature                        | °C   |
+
+#### Channel groups for Site API - Hourly Forecast Channels:
+
+| Channel                    | Description                               |
+|----------------------------|-------------------------------------------|
+| currentHoursForecast       | Current hours forecast                    |
+| currentHoursForecastPlus01 | 01 hour after the current hours forecast  |
+| currentHoursForecastPlus02 | 02 hours after the current hours forecast |
+| currentHoursForecastPlus03 | 03 hours after the current hours forecast |
+| currentHoursForecastPlus04 | 04 hours after the current hours forecast |
+| currentHoursForecastPlus05 | 05 hours after the current hours forecast |
+| currentHoursForecastPlus06 | 06 hours after the current hours forecast |
+| currentHoursForecastPlus07 | 07 hours after the current hours forecast |
+| currentHoursForecastPlus08 | 08 hours after the current hours forecast |
+| currentHoursForecastPlus09 | 09 hours after the current hours forecast |
+| currentHoursForecastPlus10 | 10 hours after the current hours forecast |
+| currentHoursForecastPlus11 | 11 hours after the current hours forecast |
+| currentHoursForecastPlus12 | 12 hours after the current hours forecast |
+| currentHoursForecastPlus13 | 13 hours after the current hours forecast |
+| currentHoursForecastPlus14 | 14 hours after the current hours forecast |
+| currentHoursForecastPlus15 | 15 hours after the current hours forecast |
+| currentHoursForecastPlus16 | 16 hours after the current hours forecast |
+| currentHoursForecastPlus17 | 17 hours after the current hours forecast |
+| currentHoursForecastPlus18 | 18 hours after the current hours forecast |
+| currentHoursForecastPlus19 | 19 hours after the current hours forecast |
+| currentHoursForecastPlus20 | 20 hours after the current hours forecast |
+| currentHoursForecastPlus21 | 21 hours after the current hours forecast |
+| currentHoursForecastPlus22 | 22 hours after the current hours forecast |
+| currentHoursForecastPlus24 | 24 hours after the current hours forecast |
+
+### Site API - Daily Forecast Channels
+
+| Channel                         | Type                  | Description                                             | Unit | MET Office Data Description                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
I would think it is better to only list the Met Office Data Description as that looks to be the most precize.

Reply:
Yeah agreed, I have removed the description originally I only had that description from the API responses, but then found a doc somewhere else that gave the technical data. I added that to this other column. As you say it negates the need for the original description so its been removed.